### PR TITLE
HARP-5188: 3D Lines buffer generation.

### PR DESCRIPTION
**Pull request by amsgames94**

### CODE REVIEW

**Comment by amsgames94** [comment]
GPU tests please

File: @here/harp-materials/lib/ShaderChunks/LinesChunks.ts [hunk 1]
```diff
@@ -5,23 +5,50 @@
...
+        pos += uv.y * lineWidth * dir / cos(angle / 2.0);
+        uv.x = ss + uv.x * lineWidth * uv.y * tan(angle / 2.0);
+    }
+    else {
+        vec3 t = vec3(-dir.y, dir.x, dir.z);
+        pos += uv.y * lineWidth * dir + uv.x * lineWidth * t;
+        uv.x = ss + uv.x * lineWidth;
+    }
+
+    /* if (angle != 0.0) {
```
Comment (robertoraggi) [inline @here/harp-materials/lib/ShaderChunks/LinesChunks.ts]: I would prefer if you could remove this commented out code.

Comment (amsgames94) [reply @here/harp-materials/lib/ShaderChunks/LinesChunks.ts]: Adressed in last revision.

**Comment by nzjony** [comment]
I notice there are no tests, I think it would be good to have some IBC tests to check we don't have any regressions in future.

**Pull request merged**